WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/All done/MigrationDoneCenterView.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+class MigrationDoneCenterView: UIView {
+
+    private lazy var iconImageView: UIImageView = {
+        let imageView = UIImageView(image: Appearance.iconImage)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.contentMode = .scaleAspectFit
+        imageView.setContentCompressionResistancePriority(.defaultLow, for: .vertical)
+        imageView.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
+        return imageView
+    }()
```

Comment:
Since this block is not calling `self`, perhaps it would be great to make this variable a constant ( `let` instead of `lazy var` ). It better communicates that this property won't be mutated.

Reply:
it is true that in this specific case, we are not getting much advantage from the lazy initialization, because that view will be there regardless. On the other hand, since the property is private and cannot be accessed from the outside of this view, I'd rather leave it as it is just to maintain a consistent style because, for example, we can't do the same on `mainStackView` and, transforming that one to a `let` constant would create overhead in the initializer.